ui/console/src/main/scripts/plugins/accounts/plugins/accounts/ts/organizations.ts

```diff
@@ -19,8 +19,8 @@
 module HawkularAccounts {
 
     export var OrganizationsController = _module.controller("HawkularAccounts.OrganizationsController", [
-        '$rootScope', '$scope', 'HawkularAccounts.OrganizationService', '$log', '$location',
-        ($rootScope, $scope, OrganizationService, $log, $location) => {
+        '$rootScope', '$scope', '$log', '$location', 'HawkularAccount',
+        ($rootScope, $scope, $log, $location, HawkularAccount) => {
 
             $scope.organizations = [];
```

Comment:
Angular best practices is not to use $scope at all. But instead, use 'this' and 'ControllerAs' syntax.
For reference: http://toddmotto.com/digging-into-angulars-controller-as-syntax/